helix-term/src/ui/picker.rs

```diff
@@ -431,7 +431,7 @@ impl<T: Item> Picker<T> {
         let mut picker = Self {
             options,
             editor_data,
-            matcher: Box::new(Matcher::default()),
+            matcher: Box::<fuzzy_matcher::skim::SkimMatcherV2>::default(),
```

Comment:
Same here

Reply:
Thanks so much @pascalkuthe for your review! 
Reverted